stanford/roles/cluster/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 CLUSTER_AMI: 'ami-07585467'
 CLUSTER_ASSIGN_PUBLIC_IP: no
+CLUSTER_EBS_SIZE: 5
+CLUSTER_INSTANCE_TYPE: db.t2.micro
```

Comment:
We shouldn't override these in `defaults`, but instead in the `db.yml` playbook.
If we override here, it'll break the majority of of plays which are already using the original defaults.

Let's move these two variables into `playbooks/db.yml`.

Reply:
db.yml will be common to all db plays. Shouldn't the vars go in db-analytics1.yml files?

Reply:
I mean, it depends...

We should presumably have some value set as the default for all databases. Given that this is the only one so far, I'd say let's make this common for all DBs.

If/when we need different values for other databases, we can decide to either override the value in the _new_, `db-whatever1.yml` plays, or then provide new DB defaults and then chose to specifically override for analyltics.

For now, the best/easiest path forward is to make these common to all DBs,
so that we avoid trying to predict the future.